docs/utils/script.py

```diff
@@ -0,0 +1,163 @@
+import os
+import sys
+import yaml
+import markdown
+
+MODULES = [
+    "ads",
+    "bin",
+    "client",
+    "organic",
+    "utils",
+]
+IGNORED_FILES = {
+    "ads.md", #Ignore this file since ads/__init__.py is converted into ads.md
+}
+PROJECT_PATH = os.path.abspath(os.getcwd())
+
+def set_up_python_path():
+    '''
+    Set up python path
+    '''
+    sys.path.append(PROJECT_PATH)
+
+
+def remove_old_doc():
+    '''
+    Remove old doc in docs/pinterest
+    '''
+    print("----------------- " + "Cleaning up old docs")
+
+    files = next(os.walk("docs/pinterest"), (None, None, []))[2]
+    for file in files: os.remove(f"docs/pinterest/{file}")
+
+
+def generate_new_doc():
+    '''
+    Use lazydoc to generate new doc at: docs/pinterest
+    '''
+    print("----------------- " + "Generating md docs")
+
+    from lazydocs import generate_docs
+    generate_docs(
+        ['pinterest'],
+        output_path='docs/pinterest/',
+        src_base_url='https://github.com/pinterest/pinterest-python-sdk/blob/main/docs/pinterest/')
+
+
+def create_file_index() -> dict:
```

Comment:
Try to modularize this function. It is very long, places where comments are used can be abstracted to helper functions inside or outside this function instead of comments.